src/components/samplePreviews/SingleSelect.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { KolImage } from '@public-ui/react';
+
+const SingleSelect = () => <KolImage _src="/assets/samples/single-select.png" _alt="" _sizes="20vw" />;
```

Comment:
Eigentlich spricht hier doch nichts dagegen, die Komponente "richtig" einzubinden, oder? So machen wir es normalerweise und so ist es auch für die Combobox schon umgesetzt.

Man kann argumentieren, dass man die Komponente mit einem Screenshot besser zeigen kann (offene Suggestions), aber in diesem Fall würde ich es lieber einheitlich mit der Combobox gestalten.